sdk/include/platform/arty-a7/platform-entropy.hh

```diff
@@ -5,7 +5,7 @@
 #include <platform/concepts/entropy.h>
 #include <riscvreg.h>
 
-DECLARE_AND_DEFINE_INTERRUPT_CAPABILITY(RevokerInterruptEntropy,
```

Comment:
This looks unrelated to other changes?

Reply:
Yup, accidentally committed and reverted.